### DISCUSSION TITLE
MGET sort of thing for Redis Hashes

### DISCUSSION THREAD

**Discussion by arun-j02** [redis/lettuce#2489] [General]
Hey, just wanted to ask like we have mget to get multiples keys-value pairs at once. Do we have something similar for getting redis hashes? UseCase: we have stored a multi level map {k1 : {f1: v1, f2, v2},  k2 : {f1: v1, f2: v2}} as redis hash. I want to pass k1, k2 as the list of keys and want to get the full data  for k1, k2 ==> {k1 : {f1: v1, f2, v2},  k2 : {f1: v1, f2: v2}}. I know redis-cli doesn't support mget sort of thing for hashes, but like does lettuce on its side might do some pipelining or something. 
Currently, we are doing this...we have a list of keys [k1, k2 ... kn] and for each key we are making a redis call. if the list is large then latencies are increasing which is obvious.

**Comment by mp911de** [comment]
Duplicate of #2490
